src/scripts/db_script.sql

```diff
@@ -168,3 +168,18 @@ CREATE TABLE order_status_histories(
         PRIMARY KEY (id)
 );
 
+-- Table : chefs 
+DROP TABLE IF EXISTS chefs;
+
+CREATE TABLE chefs(
+              id BIGSERIAL NOT NULL,
+            name VARCHAR(30) NOT NULL,
```

Comment:
Please make all the names consistent, to be the same across tables.

Also we have a users table, make this chef table to be related to users in order to re-use same fields.

Reply:
done.